db/1.9/modules/ROOT/pages/database-execute-script.adoc

```diff
@@ -0,0 +1,81 @@
+= Execute a Script Examples - Mule 4
+
+Anypoint Connector for Database (Database Connector) Execute Script operation executes a SQL script of random length as a single database statement.
+
+The Execute Script operation frequently creates schemas and tables, inserts data, and performs data rotation. Data rotation is performed by nightly jobs that move and archive data into historic tables and purge on-line tables. Some points to consider:
+
+* The script can contain multiple statements
+* Statements can be of different types
+* No input or output parameters are accepted
+* The operation runs any script that does not involve a SQL projection
+
+You can use the Execute Script operation embedding the script in the operation, or reference the script from a file, but you cannot use both ways of executing a script simultaneously. +
+
+Executing a script returns an array of integer numbers, one element per each executed statement. Each number represents the number of objects affected by the statement.
+
+The Execute Script and bulk operations are intended for different uses. For example, although you can build a script that inserts many rows into the database, you could not conveniently provide dynamic parameters to the script. Also, ensure that the script protects against a SQL injection attack.
+
+The following examples illustrates how to configure the Execute Script operation embedding the script in the operation, and how to reference the script from a file.
```

Comment:
GLOBAL: review grammar rules around subject-verb agreement and fix.
Replace "examples illustrates" with
example illustrates
OR
examples illustrate